src/kOS.Safe/Encapsulation/Suffixes/SuffixResult.cs

```diff
@@ -0,0 +1,22 @@
+namespace kOS.Safe.Encapsulation.Suffixes
```

Comment:
This is what will get used for all of the older suffixes. it is by far the simplest